src/main/java/frc/robot/systems/ElevatorWristFSM.java

```diff
@@ -194,20 +180,95 @@ private FSMState nextState(TeleopInput input) {
 	 * @param input Global TeleopInput if robot in teleop mode or null if
 	 * the robot is in autonomous mode.
 	 */
+
 	private void handleIdleState(TeleopInput input) {
-		//PREVIOUS CODE: wristMotor.set(0);
-		pidControllerWrist.setReference(currentEncoder,
-			CANSparkMax.ControlType.kPosition);
+		wristMotor.set(pid(wristMotor.getEncoder().getPosition(), currentEncoder));
+		//Zeroes the encoder at a given configuration, mainly for testing
+		if (input.isWristZeroButtonPressed()) {
+			currentEncoder = 0;
```

Comment:
I'm not seeing the need to have a separate currentEncoder variable, any reason why you can't just query the encoder object?

Reply:
When we're trying to keep the wrist in place while its in the idle state, the currentEncoder tells us what to PID to